### PR TITLE
Address points from DockerHub review

### PR DESCRIPTION
**Pull request by danxmoran**
See the review comments from docker-library/official-images#9594

### CODE REVIEW

File: influxdb/2.0/alpine/Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN addgroup -S -g 1000 influxdb && \
 # https://github.com/tianon/gosu/releases
 ENV GOSU_VER 1.12
 RUN set -eux; \
-    ARCH=$(arch) && \
+    ARCH="$(apk --print-arch)" && \
```
Comment (lesam) [inline influxdb/2.0/alpine/Dockerfile:16]: why did this change?

**Review by lesam — APPROVED** [review]
lgtm

File: influxdb/2.0/alpine/entrypoint.sh [hunk 2]
```diff
@@ -29,17 +30,18 @@ function log () {
...
-    local logtime=$(date --utc +'%FT%T.%NZ')
+    local logtime="$(date --utc +'%FT%T.%NZ')"
     1>&2 echo -e "${logtime}\t${level}\t${msg}\t{${attrs}}"
 }
 
 # Set the global log-level for the entry-point to match the config passed to influxd.
 function set_global_log_level () {
-    local level="$(influxd print-config --key-name log-level ${@})"
+    local level="$(influxd print-config --key-name log-level "${@}")"
     if [ -z "$(log_level ${level})" ]; then
```
Comment (yosifkit) [inline influxdb/2.0/alpine/entrypoint.sh:40]: This will never be false, right? `log_level` always returns a string.

Comment (danxmoran) [reply influxdb/2.0/alpine/entrypoint.sh:40]: Good catch (mirrored the non-alpine version, but didn't clean up)

**Pull request merged**